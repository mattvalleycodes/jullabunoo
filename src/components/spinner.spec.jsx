import * as React from "react";
import renderer from "react-test-renderer";

import Spinner from "./spinner";

describe("<Spinner /> component", () => {
  test("renders a spinner", () => {
    const spinner = renderer.create(<Spinner />).toJSON();

    expect(spinner).toMatchSnapshot();
  });
});
