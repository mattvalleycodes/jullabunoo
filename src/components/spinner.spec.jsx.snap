// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> component renders a spinner 1`] = `
<div>
  Loading ...
</div>
`;
